setup.cfg

```diff
@@ -43,6 +43,7 @@ install_requires =
     asgiref >= 3.2
     pytz
     sqlparse >= 0.2.2
+    colorama; sys_platform == "win32"
```

Comment:
Please keep this list sorted alphabetically.

Reply:
done!